src/shared/components/data-table/components/table-controls/components/table-actions/components/popup-panel/PopupPanel.scss

```diff
@@ -0,0 +1,11 @@
+@import 'src/styles/common';
+
+.popupPanel {
+  display: flex;
+  background-color: $light-grey;
+  position: absolute;
+  z-index: 2;
```

Comment:
`2` -> `1`? (if no z-index on header)

Reply:
Looks like I need z-index on header for now